raiden/routing.py

```diff
@@ -24,6 +26,38 @@ def get_best_routes(
         to_address: typing.TargetAddress,
         amount: int,
         previous_address: typing.Optional[typing.Address],
+        config: Dict,
+) -> List[RouteState]:
+    services_config = config.get('services', None)
+
+    if services_config and services_config['pathfinding_service_address'] is not None:
+        return get_best_routes_pfs(
```

Comment:
The previous routing mechanism should be used as a fallback in case the PFS doesn't return a valid response for some reason.